example/.camunda/element-templates/http-json-connector.json

```diff
@@ -7,6 +7,10 @@
     "appliesTo": [
       "bpmn:ServiceTask"
     ],
+    "category": {
+      "id": "rest",
+      "name": "REST templates"
+    },
```

Comment:
Will that live outside of the JSON Schema and only be used for the extension?

Reply:
No, that should be part of our official element template schema, if we decide to go down this route.

Reply:
👍

Reply:
Opened an issue: https://github.com/bpmn-io/bpmn-js-properties-panel/issues/633.